src/components/widgets/Number.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+
+import Header from './Header'
+
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+export default class Number extends Component {
+  calculateProgress() {
+    const { statistic, goal } = this.props
+    return ( statistic / goal ) * 100
+  }
+
+  roundTwoDecimals( data ) {
+    return parseFloat( data ).toFixed( 2 )
+  }
+
+  calculateProgressPercentage() {
+    return this.roundTwoDecimals( this.calculateProgress() )
+  }
+
+  render() {
+    const { title, x, y, statistic, goal } = this.props
+    const positionStyle = { top: x, left: y }
+
+    const progress = this.calculateProgressPercentage()
+
+    const isComplete = statistic / goal >= 1
+
+    return (
```

Comment:
I think it would be cool to split this component into two. Have a NumberContainer, and a Number component. The NumberContainer has the methods and then renders only the number. The number is then strictly presentational and the container deals with the data manipulation. Better separation of concerns and awesome encapsulation!